src/util/SpawnPed.cpp

```diff
@@ -1,48 +1,79 @@
-#include "core/frontend/Notifications.hpp"
-#include "game/backend/FiberPool.hpp"
-#include "game/backend/ScriptMgr.hpp"
-#include "game/rdr/Natives.hpp"
-
+#include "SpawnPed.hpp"
 
 namespace YimMenu
 {
-	bool SpawnPed(std::string model_name, Player player, int playerPed)
+	extern int SpawnPed(std::string model_name, Player player, int playerPed, bool blockNewPedMovement, bool spawnDead, bool invincible, bool invisible, int scale)
```

Comment:
I know this is unexpected as ive merged these changes before. But a base feature such as SpawnPed has to be as minimal as possible to allow for flexibility. I see no use in having a player index and a player ped as params. I suggest you refactor this.